pkg/githubutils/githubutils.go

```diff
@@ -0,0 +1,74 @@
+package githubutils
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"os"
+
+	"github.com/grafana/plugin-validator/pkg/logme"
+)
+
+func FetchLatestGrafanaSdkRelease() (Release, error) {
+	return FetchGrafanaSdkReleaseByTag("latest")
+}
+
+func FetchGrafanaSdkReleaseByTag(tag string) (Release, error) {
+	if tag == "" {
+		return Release{}, fmt.Errorf("tag is required")
+	}
+
+	apiUrl := "https://api.github.com/repos/grafana/grafana-plugin-sdk-go/releases"
```

Comment:
Nit: This can be a `const`:
```suggestion
	const apiUrl = "https://api.github.com/repos/grafana/grafana-plugin-sdk-go/releases"
```